jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -327,18 +331,34 @@ default long getLength()
         void demand(Runnable demandCallback);
 
         /**
-         * <p>Fails this content source, possibly failing and discarding accumulated
-         * content chunks that were not yet read.</p>
+         * <p>Fails this content source with a terminal {@link Chunk#getError() error},
+         * possibly failing and discarding accumulated content chunks that were not yet read.</p>
```

Comment:
_possibly failing and discarding_ is unclear since that is guaranteed. I would just get rid of the _possibly_ word.